triggers/incident-triggered/trigger.yaml

```diff
@@ -40,6 +40,20 @@ examples:
           title: !Data title
           urgency: !Data urgency
           serviceName: !Data serviceName
+- summary: When a PagerDuty incident is triggered
+  content:
+    apiVersion: v1
+    kind: Trigger
+    name: pagerduty-incident
+    source:
+      type: webhook
+      image: relaysh/pagerduty-trigger-incident-triggered
+      binding:
```

Comment:
@kenazk this is indented 1 tab too far. `binding` should be a sibling of source.